### ISSUE TITLE
Get intermediate layer results

### ISSUE THREAD

**Issue by andrewcby** [microsoft/unilm#145]
Is there a way to get intermediate results from the model? Looks like I'm only able to get the last layer of the network (the logits)

**Comment by nyanyanya** [comment]
Hi @andrewcby,

Unfortunately there is no command line or APIs for that. You have to check and modify code at src/pytorch_pretrained_bert/modeling.py to get the intermediate results.

Thanks,
Nan

**Comment by wolfshow** [comment]
@andrewcby I am afraid you have to modify the code yourself.
